Resources/public/mapbender.element.digitizer.js

```diff
@@ -1285,22 +1265,40 @@
         /**
          * Copy feature
          *
-         * @param feature
+         * @param {OpenLayers.Feature} feature
          */
-        copyFeature: function(feature) {
+        copyFeature: function(feature ) {
             var widget = this;
             var schema = feature.schema;
+            var layer = feature.layer;
             var newFeature = feature.clone();
             var config = schema.copy;
             var defaultAttributes = getValueOrDefault(config, "data",{});
+            var allowCopy = true;
 
-            newFeature.attrubites = _.extend({}, defaultAttributes, feature.attributes);
-            _.each(defaultAttributes,function(key,value) {
-               if(newFeature.attrubites[key] === null){
-                   newFeature.attrubites[key] = value;
-               }
+            _.each(schema.copy.rules, function(ruleCode) {
+                var f = feature;
```

Comment:
unused variable

Reply:
@sveta-wl, used by the code evalutation.